deploy-manage/upgrade/deployment-or-cluster/upgrade-on-ece.md

```diff
@@ -3,3 +3,46 @@ navigation_title: "Upgrade on {{ece}}"
 ---
 
 # Upgrade on {{ece}} (ECE)
+
+Similar to {{ech}}, a single click in the {{ecloud}} console can upgrade a deployment running on ECE to a newer version, add more processing capacity, change plugins, and enable or disable high availability, all at the same time. During the upgrade process, {{es}}, {{kib}}, and all of your deployment components are upgraded simultaneously.
+
+Once you're prepared to upgrade, do the following: 
+
+% Note: Add a link once confirmed where prepare to upgrade will reside in TOC. 
+
+1. Ensure your current ECE and Docker or Podman versions are [compatible](https://www.elastic.co/support/matrix/#elastic-cloud-enterprise) with the {{stack}} version you're upgrading to. For example, if you're upgrading to 9.0.0, the minimum required version is ECE 3.0. If you don’t have a compatible version installed, [upgrade your orchestrator](/deploy-manage/upgrade/orchestrator/upgrade-cloud-enterprise.md).  
+2. Download the most recent [stack pack](/deploy-manage/deploy/cloud-enterprise/manage-elastic-stack-versions.md#ece_most_recent_elastic_stack_packs) for the version you’re upgrading to, then [add the stack pack](/deploy-manage/deploy/cloud-enterprise/manage-elastic-stack-versions.md#ece-manage-elastic-stack-add) to your installation using the Cloud UI. 
+3. [Configure a snapshot repository](/deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md) to enable snapshots and back up your data.
```

Comment:
I would mark this as optional, in order to upgrade there's no need to configure a repo, and in a lot of systems this will already be configured for the deployment, so the step could just cause confusion.

I would state with something like:
```suggestion
3. If not configured already, [assign a snapshots repository](/deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md) to your deployment, to enable snapshots and back up your data. This is an optional but recommended step.
```